Gems/O3DE/GeomNodes/Code/Source/Editor/Common/GNAPI.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Editor/Common/GNAPI.h>
+
+namespace GeomNodes
+{
+    namespace API
+    {
+        bool Init(u64 id, HandlerCallback cb)
+        {
+#ifdef USE_BRIDGE
+            return ::Init(id, cb);
+#else
+            (void)id;
+            (void)cb;
+            return false;
+#endif
```

Comment:
Consider using the attribute `[[maybe_unused]]` on the function parameters.
```
bool Init([[maybe_unused]] id, [[maybe_unused]] HandlerCallback cb)
{
#ifdef USE_BRIDGE
            return ::Init(id, cb);
#else
            return false;
}
```
There are multiple methods in this file that could do the same

Reply:
Gotcha. I'm using a mixed of both in my code so I'll review the other instances as well and use the attribute.

Reply:
fixed in commit [d756b38](https://github.com/o3de/o3de-technicalart/pull/11/commits/d756b38541172fd3e0051140ca96cc9bd8b05388)